src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -658,13 +668,18 @@ public void Initialize()
 
             options.OnIntegrityErrorOfAlreadySyncedData += (obj, e) =>
             {
+                string title = "Integrity error of already synced data - System Storage";
+
+                if (Logger.IsOperationsEnabled)
+                    Logger.Operations($"{title}. {e.Message}", e.Exception);
+
                 var alert = AlertRaised.Create(
                     null,
-                    "Integrity error of already synced data - System Storage",
+                    title,
                     e.Message,
                     AlertType.IntegrityErrorOfAlreadySyncedData,
                     NotificationSeverity.Warning,
-                    "Integrity Error of Synced Data - System");
+                    key: $"Integrity Error of Synced Data - System/{SystemTime.UtcNow.Ticks % 5}"); // if this was called multiple times let's try to not overwrite previous alerts
```

Comment:
If the error is the same (same db, etc), even on a minute basis, we'll still create a lot of alerts, no?

Reply:
We can use the db path for distinguishing between them?

Reply:
This one is only for System storage

Reply:
For db / indexes we distinguish using db name and storage type (documents, index, configuration)